app/templates/emails/digital-outcomes-and-specialists-2_application_started.html

```diff
@@ -6,12 +6,30 @@
 <body>
     Dear supplier,
     <br /><br />
-    You, or one of your colleagues, has started an application for the Digital Outcomes and Specialists 2 framework. All contributors on your Digital Marketplace account will now receive email updates.
+    You, or one of your team, started an application to become a Digital Outcomes and Specialists 2 supplier.
+    All contributors on your Digital Marketplace account will now receive email updates.
+    The deadline for completing your application is 6 December 2016.
     <br /><br />
-    You need to confirm that you’re eligible to offer services under Digital Outcomes and Specialists 2 as part of the application. You don’t need to describe your services now. You’ll be asked to provide more details when you respond to a buyer’s specific brief.
+    You must ask any clarification questions you have through your Digital Marketplace account before
+    5pm GMT, 22 November 2016.
```

Comment:
Ultimately these dates should probably be pulled from the content loader I guess.